### DISCUSSION TITLE
Does Flare have a combat log?

### DISCUSSION THREAD

**Discussion by WithinAmnesia** [flareteam/flare-game#925] [Q&A] [answered]
I am like going bonkers thinking Flare has like a combat log or something to check what happened recently but I can't find like a keybind for it and I wonder if Flare even had one to begin with lol. https://github.com/flareteam/flare-engine/wiki/Attribute-Reference#inputstate-default-keybindings

**Comment by dorkster** [comment, marked as answer]
There is no combat log. I don't think such a thing really has a place in an ARPG like Flare.
